tests/helpers/test_framework_helpers.py

```diff
@@ -186,6 +203,7 @@ def test_find_suppliers_with_details_and_draft_service_counts(mock_data_client):
                 'onFramework': True,
                 'countersignedPath': 'some/path',
                 'countersignedAt': '2017-01-02T03:04:05.000006Z',
+                'agreementId': None
```

Comment:
Same as above: how can it be countersigned but have an `agreementId` of `None`?
I know it's not being tested, but I suggest setting it to `31385` just to ensure the test data is correct.

Reply:
fixed in ab1083c